README.md

```diff
@@ -27,9 +27,9 @@ Project source code: [https://github.com/jarviscanada/jarvis_data_eng_AndresOsor
 
 **Data Wrangling** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AndresOsorio/tree/master/python_data_wrangling)]: Implemented a data analysis project to help a retail organization drive their revenue back up; specifically, the analysis segmented customers (using RFM Segmentation) into important categories to help the company develop targeted marketing strategies. Used a PostgreSQL instance as a data warehouse and Python Pandas/Jupyter to perform OLAP operations on the data and answer necessary business questions.
 
-**Hadoop** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AndresOsorio/tree/master/hadoop)]: In Progress
+**Hadoop** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AndresOsorio/tree/master/hadoop)]: Evaluated Hadoop core components (HDFS, YARN, and MapReduce), deployed a 3-node Hadoop cluster using Google Cloud Platform Dataproc, and answer several business questions using Hive and Zeppelin; leveraged the distributed storage and processing capabilities of the Hadoop ecosystem to analyze the 2016 World Development Indicators dataset and solve the required business problems.
```

Comment:
~answer several business questions~

did data analytics using Hive and Zeppelin